usaspending_api/awards/v2/filters/filter_helpers.py

```diff
@@ -42,7 +47,51 @@ def date_or_fy_queryset(date_dict, table, fiscal_year_column, action_date_column
     return False, None
 
 
-def total_obligation_queryset(amount_obj, model):
+def sum_transaction_amount(qs, aggregated_name='transaction_amount', filter_types=award_type_mapping,
+                           calculate_totals=True):
+    """ Returns correct amount for transaction if loan (07, 08) vs all other award types (covers IDV)"""
+    aggregate_dict = {}
+    if calculate_totals:
+        qs = qs.annotate(total_subsidy_cost=Sum(Case(When(type__in=list(loan_type_mapping),
+                                                          then=F('original_loan_subsidy_cost')),
+                                                     default=0,
+                                                     output_field=FloatField())))
```

Comment:
Do we need to set the precision here? with us specifying in the model the precision to be 2 decimal places, there shouldn't be a violation, but if it's possible to also explicitly put the precision here, we may want to do that.

Reply:
Output field was not casting fields within SQL query. Removing `output_field` from annotation since fields are already decimal w/ precision 2.  Tested it locally and it did not cause an error. Performance was not affected.